src/modules/Dashboard/api/api.ts

```diff
@@ -395,6 +396,7 @@ export const getReviewActivitiesApi = (
     params: {
       respondentId,
       createdDate,
+      limit: MAX_LIMIT,
```

Comment:
to avoid using max limit, it's better to add infinite scroll here and in other cases

something like in this PR: https://github.com/ChildMindInstitute/mindlogger-admin/pull/1072

if you need, pls contact with Aleksandr to get more details